doc/changelog.md

```diff
@@ -157,78 +293,72 @@
   [{'expval0': (array(-0.0978434), array(-0.19767681))}, (array(-0.0978434), array(-0.19767681))]
   ```
 
-* Add native support for `qml.PSWAP` and `qml.ISWAP` gates on Amazon Braket devices. Specifically, a circuit like
-
-  ```py
-  import pennylane as qml
-  from catalyst import qjit
+* Support has been added for linear algebra functions which depend on computing the eigenvalues
+  of symmetric matrices, such as `np.sqrt_matrix()`.
```

Comment:
Are there additional numpy functions that we know about in this category?

Reply:
@maliasadi?

Reply:
@dime10 @josh146 I've tested the support with `np.eig` and `np.eigvals`. But there can be more linear algebra methods that depend on the calculation of eigenvalues under the hood.